designs/2022-docs-information-architecture-update/README.md

```diff
@@ -0,0 +1,358 @@
+- [NEW PAGE] Report Security Vulnerabilities
+  - Takes current content from the [Reporting a Security Vulnerability section on the Contributing page](https://eslint.org/docs/latest/developer-guide/contributing/#reporting-a-security-vulnerability). Worth breaking out into its own page so that content not only on the hidden landing page and for discoverability and SEO for an important subject.
+  - NOTE: Should be added, but not part of the initial IA project. Probably can be done as a one-off task or group in with one of the project phases.
+- [MOVE PAGE] Governance
+  - Current page: [Governance](https://eslint.org/docs/latest/maintainer-guide/governance)
+
+### [REFACTOR SECTION] Maintain ESLint
+
+- [NO CHANGE] Maintain ESLint section landing page
+  - Current page: [Maintainer Guide](https://eslint.org/docs/latest/maintainer-guide)
+  - Hidden in sidebar
+- [MOVE & RENAME PAGE] Manage Issues
+  - Current page: [Managing Issues](https://eslint.org/docs/latest/maintainer-guide/issues)
+- [MOVE & RENAME PAGE] Review Pull Requests
+  - Current page: [Reviewing Pull Requests](https://eslint.org/docs/latest/maintainer-guide/pullrequests)
+- [MOVE & RENAME PAGE] Manage Releases
+  - Current page: [Managing Releases](https://eslint.org/docs/latest/maintainer-guide/releases)
+- [NEW PAGE(S)] Any other technical pages for maintainers
+  - I don’t have any thoughts on what these should be right now. I would defer to the core maintainers on what they think other maintainers should know.
+  - I think that these additional technical pages would make sense to include in **Phase 5: “Maintain ESLint” documentation updates**of [the documentation update project](https://github.com/eslint/eslint/issues/16365).
+  - NOTE: Maybe should be added, but not part of the initial IA project.
+- [MOVE PAGE] Run the Tests
+  - Current page: [Unit Tests](https://eslint.org/docs/latest/developer-guide/unit-tests)
+
+### New Sidebar (summary)
+
+With the above changes, the left-hand side navigation will look like:
+
+- Use ESLint in Your Project (landing page)
+  - Getting Started
+  - Core Concepts
+  - Configure
+    - Configuration Files (New)
+    - Configuration Files
+    - Configure Language Options
+    - Configure Rules
+    - Configure Plugins
+    - Ignore Code
+  - Command Line Interface Reference
+  - Rules Reference
+  - Integrations
+  - Migrate to v8.x
+- Extend ESLint (landing page)
+  - Create Plugins
+  - Custom Rules
+  - Custom Formatters
+  - Custom Parsers
+  - Custom Processors
+  - Share Configurations
+  - Node.js API Reference
+- Contribute to ESLint (landing page)
+  - Report Bugs
+  - Propose a New Rule
+  - Propose a Rule Change
+  - Request a Change
+  - Architecture
+  - Set up a Development Environment
+  - Run the Tests
+  - Work on Issues
+  - Submit a Pull Request
+  - Code of Conduct
```

Comment:
Perhaps "Code of Conduct" should be the first item under "Contribute to ESLint", as it's now on the [contributing](https://eslint.org/docs/latest/developer-guide/contributing/) page?